CppHeaderParser/CppHeaderParser.py

```diff
@@ -377,6 +377,37 @@ def filter_out_attribute_keyword(stack):
         return stack
 
 
+_nhack = re.compile(r"[A-Za-z_][A-Za-z0-9_]*")
```

Comment:
Could probably just use `\w` for the second class here.

Reply:
The other regexes in here aren't \w, so I elected to copy those. I think I'd rather fix them all at the same time, some other time.